Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -316,6 +324,12 @@ namespace AzToolsFramework
                     auto handler = dpeSystem->CreateHandlerInstance(handlerId);
                     handler->SetValueFromDom(childValue);
                     addedWidget = handler->GetWidget();
+
+                    // only set the widget's tooltip if it doesn't already have its own
+                    if (!descriptionString.empty() && addedWidget->toolTip().isEmpty())
+                    {
+                        addedWidget->setToolTip(QString::fromUtf8(descriptionString.data()));
+                    }
```

Comment:
Does this mean that the row widget and its child's handler widget could have the same tooltip? Not that it's necessarily wrong; just trying to sort it out.

Reply:
yes, correct. That's going to be the typical result, actually. If you check the RPE for a typical asset in the AssetEditor, that's what it does now.